### ISSUE TITLE
Angular 5.1.x update missing @angular/service-worker/build/gulp

### ISSUE THREAD

**Issue by garrettH3S** [NathanWalker/angular-seed-advanced#478]
This module no longer contains this method. How can I update this file to work with angular 5.1.x?

https://github.com/NathanWalker/angular-seed-advanced/blob/2d652e157a4841830f616fbc9be69d8fddf2ae2e/tools/tasks/seed/sw.manifest.static.ts#L2

**Comment by m-abs** [comment]
Did you find a solution? I'm having the same problem?

**Comment by m-abs** [comment]
I'm answering myself:
I downloaded https://github.com/angular/mobile-toolkit/blob/master/service-worker/worker/src/build/gulp.ts placed it in tools/utils, made a few minor fixes to make typescript accept it.

**Comment by garrettH3S** [comment]
@m-abs I haven't found a solution yes. I removed the gulp job in the interim, I assume you cloned the mobile-toolkit down and imported it?

**Comment by m-abs** [comment]
Yes, just that one file. Typescript complaint about a few things, that I fixed.

I probably could just have deleted the task, we don't use service-workers (yet).
